datasource/src/test/scala/quasar/physical/mongo/interpreter/MaskSpec.scala

```diff
@@ -51,9 +51,23 @@ class MaskSpec extends Specification with quasar.TreeMatchers {
     Mask[InState](masks) run state map (_ leftMap (_.mapper))
 
   "state modifications" >> {
-    "unfocused" >> {
+    "unfocused non root" >> {
+      val state = InterpretationState("unique", Mapper.Unfocus)
+      val result = eval(state, Map(CPath.parse("foo") -> Set(ColumnType.String)))
```

Comment:
@cryogenian @jsantos17 I was curious why this particular test was added if the problematic query was `select a, b, from data`?
```
(v93.1.57-372dd24) 💪 $ select foo, bar from data                                             
QScript (Optimized):
InterpretedRead(ResourcePath(/datasource/194fd977-5a22-4a5c-aea4-d5caa90cbd16/data))
╰─ ScalarStages(ExcludeId)
   ╰─ Mask
      ├─ Column[.foo](⊤)
      ╰─ Column[.bar](⊤)
```
Should we also test `ColumnType.Top` here? Or was the bug more specific to `ColumnType.String`?

Reply:
@alissapajer it checks that result of masking is wrapped into uuid, next stages suppose that it is wrapped (`focus[F]` says them to suppose this).